custom_components/alexa_media/sensor.py

```diff
@@ -15,6 +15,7 @@
     DEVICE_CLASS_TIMESTAMP,
     STATE_UNAVAILABLE,
     TEMP_CELSIUS,
+    PERCENTAGE,
```

Comment:
Black should have sorted this alphabetically. Did you run black?

Reply:
Sorry total newb. I dont even know what black is? I added this manually. Didnt use anything to sort it.

Reply:
Black is a style enforcer for python. I guess it's tied to the fact you're not using git, but the repo is set up to run a bunch of tests/enforcements before any git commit to enforce standards. I can run it for you before we accept the PR.

Reply:
Handled by precommit.